### ISSUE TITLE
[Benchmark] Invalid Data

### ISSUE THREAD

**Issue by Cypher-Bruce** [hezean/sustc#22]
### Describe your question

When running on the benchmark data from [DB Project Benchmark](http://172.18.34.154/), the data(zip SHA256: 8445ee4a1fa5786766b91dfde31a9c1e3f3ef68a8b4e6e8c62e9e0699c93a5d7) seems to violate some constraints about user.
- User name should be unique
- User birthday should not be null or empty
<img width="400" alt="ScreenShot" src="https://github.com/hezean/sustc/assets/53269096/82d646cc-aeab-4901-9a94-e12b6f0d45cd">

After I removed the constraints, the data in my table is as shown above.
It seems that some user names cannot be recognized by DataGrip, and birthday is empty for some users.

**Comment by sses7757** [comment]
@Cypher-Bruce
1. The `name` column is not unique, as in project 1. Hence, the register API and user record documentation is fixed now.
2. The `birthday` column can be null or empty as stated in the register API documentation. Please refer to [line 19](https://github.com/hezean/sustc/blob/e05998216cbf1c1efa97d85342c135d1156c670b/sustc-api/src/main/java/io/sustc/service/UserService.java#L20).

**Comment by Cypher-Bruce** [comment]
Thank you for your reply. Now I understand about the `name` column. But the description for `birthday` column is still confusing to me.
As a corner case stated in [line 19](https://github.com/hezean/sustc/blob/e05998216cbf1c1efa97d85342c135d1156c670b/sustc-api/src/main/java/io/sustc/service/UserService.java#L19) in UserService.java:

```
<li>{@code birthday} in {@code req} is valid (not null nor empty) while it's not a birthday (X月X日)</li>
```

My understanding is that `birthday` should not be null nor empty(so it can be called valid), and it should also be in the form of "X月X日".
Additionally, it seems that some `birthday` data shown in the screenshot above violate the format. (written as "12-08")

**Comment by sses7757** [comment]
> @sses7757 Thank you for your reply. Now I understand about the `name` column. But the description for `birthday` column is still confusing to me. As a corner case stated in [line 19](https://github.com/hezean/sustc/blob/e05998216cbf1c1efa97d85342c135d1156c670b/sustc-api/src/main/java/io/sustc/service/UserService.java#L19) in UserService.java:
> 
> ```
> <li>{@code birthday} in {@code req} is valid (not null nor empty) while it's not a birthday (X月X日)</li>
> ```
> 
> My understanding is that `birthday` should not be null or empty(so it can be called valid), and it should also be in the form of "X月X日". Additionally, it seems that some `birthday` data shown in the screenshot above violate the format. (written as "12-08")

@Cypher-Bruce 
This is a statement (`A && B` is true in Java) of corner case, hence, its corresponding normal case is `A && B` is false, which is `A & !B` **or `!A`**. Besides, the corrected sample import data is now available at [172.18.34.154](http://172.18.34.154/).

**Comment by Cypher-Bruce** [comment]
Thank you for your reply.
